hpcm/hpe-mlds-build-hpcm-os-image.md

```diff
@@ -93,18 +99,23 @@ resource_pools:
   - pool_name: aux-pool
     max_aux_containers_per_agent: 100
 EOF
+chmod 640 /etc/determined/master.yaml
```

Comment:
To protect the database password in `/etc/determined/master.yaml`

Reply:
No group specified?